src/rpc/rawtransaction.cpp

```diff
@@ -1072,6 +1072,11 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
                         {
                             {RPCResult::Type::STR_HEX, "txid", "The transaction hash in hex"},
                             {RPCResult::Type::BOOL, "allowed", "If the mempool allows this tx to be inserted"},
+                            {RPCResult::Type::NUM, "vsize", "Virtual transaction size as defined in BIP 141"},
```

Comment:
I've dropped the "as defined in BIP 141" portion of the description but have retained `vsize` as `getrawmempool` [already prints out `vsize`](https://github.com/dashpay/dash/blob/10312f7d9e07e7390c14a76c16753d2921c49dc6/src/rpc/blockchain.cpp#L550) and a bunch of functional tests already refer to it like [`mempool_updatefromblock`](https://github.com/dashpay/dash/blob/10312f7d9e07e7390c14a76c16753d2921c49dc6/test/functional/mempool_updatefromblock#L89) and [`mining_prioritisetransaction`](https://github.com/dashpay/dash/blob/10312f7d9e07e7390c14a76c16753d2921c49dc6/test/functional/mining_prioritisetransaction#L65).

It's practically just an alias to `size` at this point but as long as somewhere something tries to fetch `vsize`, it's better there than not. Though on the other hand some RPCs _might_ have already purged `vsize` since I don't see them outside of the examples I've mentioned so maybe there's merit to removing them.

Regardless, it'd have to be done in a distinct commit since we'd have to regularize other `vsize` usage.